src/Tester/FSharpGrainTests.cs

```diff
@@ -14,20 +12,9 @@ namespace UnitTests.General
     /// Unit tests for grains implementing generic interfaces
     /// </summary>
     [TestClass]
-    public class FSharpGrainTests : UnitTestSiloHost
+    public class FSharpGrainTests : UnitTestSiloHostEnsureDefaultStartedSecondaryOff
```

Comment:
I'm all up for a better name. Nevertheless, not sure if "Single" is the right word. I tried to imply that I started the silo host with the default configuration (with no providers that depend on Azure, no weird config settings, etc). Currently it's not even a single silo, but 2 (although that might change if I see that it speeds up things significantly).
Can you recommend a better name, given the explanation?